k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
This is my public docker hub prefix, and should have the latest build. We'll need someone else to push to any other repository as I only have access to my own.

Until such a time, this is a valid value for anybody to use.